scripts/supercollider/unlink-programdata.ps1

```
@@ -0,0 +1,21 @@
+# Ensure the current ExecutionPolicy
+
+Set-ExecutionPolicy -Scope Process ByPass
+
+# Check Admin
+
+if (-Not ([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)) {
+    Write-Host "You NEED ADMIN permisson to remove the programdata symlink." -ForegroundColor DarkRed
```

PSScriptAnalyzer (1.24.0)

[warning] File 'unlink-programdata.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)